data/lang/cargo/en.json

```diff
@@ -0,0 +1,326 @@
+{
+   "DRIVE_CLASS1_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 1 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS2_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 2 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS3_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 3 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS4_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 4 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS5_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 5 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS6_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 6 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS7_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 7 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS8_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 8 hyperdrive (packaged)"
+   },
+   "DRIVE_CLASS9_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 9 hyperdrive (packaged)"
+   },
+   "DRIVE_MIL1_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 1 military drive (packaged)"
+   },
+   "DRIVE_MIL2_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 2 military drive (packaged)"
+   },
+   "DRIVE_MIL3_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 3 military drive (packaged)"
+   },
+   "DRIVE_MIL4_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 4 military drive (packaged)"
+   },
+   "DRIVE_MIL5_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 5 military drive (packaged)"
+   },
+   "DRIVE_MIL6_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 6 military drive (packaged)"
+   },
+   "DRIVE_MIL7_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 7 military drive (packaged)"
+   },
+   "DRIVE_MIL8_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 8 military drive (packaged)"
+   },
+   "DRIVE_MIL9_PACKAGED" : {
+      "description" : "",
+      "message" : "Class 9 military drive (packaged)"
+   },
```

Comment:
What are all these string from row 1 down to here? `DRIVE_<...>_PACKAGED`? I don't see them used in the code anywhere?

Reply:
I have code that uses them in my own branch

Reply:
But is there any reason we want to have strings that are unused in our code, at master-side?

If you need those strings for some feature on your side, then best have those in their own local branch, with that feature.